core/io/file_access_pack.cpp

```diff
@@ -284,16 +284,22 @@ bool PackedSourcePCK::try_open_pack(const String &p_path, bool p_replace_files,
 	if (enc_directory) {
 		Ref<FileAccessEncrypted> fae;
 		fae.instantiate();
+#ifdef DEBUG_ENABLED
 		ERR_FAIL_COND_V_MSG(fae.is_null(), false, "Can't open encrypted pack directory.");
+#endif
 
 		Vector<uint8_t> key;
 		key.resize(32);
 		for (int i = 0; i < key.size(); i++) {
 			key.write[i] = script_encryption_key[i];
 		}
 
+#ifdef DEBUG_ENABLED
 		Error err = fae->open_and_parse(f, key, FileAccessEncrypted::MODE_READ, false);
 		ERR_FAIL_COND_V_MSG(err, false, "Can't open encrypted pack directory.");
+#else
+		fae->open_and_parse(f, key, FileAccessEncrypted::MODE_READ, false);
```

Comment:
You should probably still check the error and refuse to set `f - fae` if it fails, just without the macro